src/transactions/actions.ts

```diff
@@ -59,6 +60,11 @@ export interface TransactionConfirmedAction {
   receipt: CeloTxReceipt
 }
 
+export interface TransactionConfirmedViemAction {
+  type: Actions.TRANSACTION_CONFIRMED_VIEM
+  txId: string
+}
```

Comment:
Made a separate action instead of reusing the existing one as the viem receipt isn't directly serializable without a custom serialization for big ints. The actual receipt isn't actually used anywhere from the redux action other than to check for success status, which is redundant since this action is only fired when the tx is a success.